elements/pl-drawing/pl-drawing.mustache

```diff
@@ -6,21 +6,19 @@
 {{#parse_error}}
 <script>
     $(function(){
-        $('#drawing-interface-{{uuid}} [data-toggle="popover"]').popover({
+        $('#pl-drawing-{{uuid}} [data-toggle="popover"]').popover({
             container: 'body',
             template: '<div class="popover pl-number-input-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
         });
     });
 </script>
-<span id="drawing-interface-{{uuid}}">
-    <a role="button" class="btn btn-light btn-sm" data-placement="auto"
-       data-trigger="focus" data-toggle="popover" data-html="true" title="Error"
-       tabindex="0" data-content="{{parse_error}}">
-        INVALID <span><i class="fa fa-question-circle" aria-hidden="true"></i></span></a>
-</span>
+<span id="pl-drawing-{{uuid}}">
+    <span class="badge text-danger badge-invalid"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid</span>
+    <a href="javascript:void(0);" role="button" class="btn btn-sm btn-secondary small border" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" style="margin-left: 5px" title="Format Error" tabindex="0" data-content="{{parse_error}}"> Why <i class="fa fa-question-circle" aria-hidden="true"></i>
```

Comment:
Why do we have `javascript:void(0);` here? I seem to recall that this was also used in a few places in the PL `.ejs` files but it was unnecessary, and not recommended by the bootstrap docs? Also, any chance we could use a class like `ml-3` rather than explicit margins? (see https://getbootstrap.com/docs/4.6/utilities/spacing/)

Reply:
Good catch, this was in the other elements and I just copy+pasted it over but I agree it can be re-worked.  I've updated it here to be a button, but we should probably stick this into an issue or something.